CONTRIBUTING.md

```diff
@@ -104,7 +104,11 @@ You can specify the special `test` attribute in the configuration that will not
 
 - `test.runNvim`: Set to `false` to avoid launching nvim with this configuration and simply build the configuration.
 
-The tests are then runnable with `nix flake check --all-systems`.
+A single test can be run with `nix develop --command tests --interactive`. This launches the testing suite in interactive mode, allowing you to easily search for and select specific tests to run.
```

Comment:
Depending on how much noise it adds, it could be good to draw attention to this part of the document to make it more obvious how to run a single test.

Reply:
Agreed, updated!